.changeset/ten-singers-rhyme.md

```diff
@@ -0,0 +1,13 @@
+---
+'xstate': major
+---
+
+The `interpret(...)` function has been deprecated and renamed to `createActor(...)`:
+
+```diff
+-import { interpret } from 'xstate';
++import { createActor } from 'xstate';
```

Comment:
just a note: it might be confusing that we have `createActor` and `createMachine` but they don't actually do a similar thing whereas things that do similar things to `createMachine` don't share the same naming scheme (`createMachine`, `fromPromise`, etc)

Reply:
Yeah perhaps, but the alternatives aren't really better:

- `createMachine` to `fromMachine` = we're creating a machine from a machine...?
- `fromMachineConfig` is a bit too wordy
- `createMachineFromPromise` wordy and confusing
- `createPromise` we're not creating a promise though